### PR TITLE
Vite

### PR DESCRIPTION
**Pull request by royduin**

### CODE REVIEW

**Comment by royduin** [comment]
Why are the Dusk tests failing?

**Comment by indykoning** [comment]
It seems almost random wether the dusk tests succeed or fail during testing. Is there a chance that the pipeline does not wait for the indexers to have finished and thus the table might or might not exist depending on the speed of the indexer?

**Comment by indykoning** [comment]
I've found the problem. We're too quick with vite! 🎉 
The flat indexer is not done by the time the styling is compiled and the tests are executing. I'll have to build in a step to wait until the flat table is available to us

**Comment by indykoning** [comment]
Tests are now functioning properly after adding a wait until the catalog_product_flat_1 table to become available.
The only failing test seems to be incorrect as the screenshot in the artifact does in fact contain "Partytime!"

File: package.json [hunk 1]
```diff
@@ -1,48 +1,37 @@
...
+        "prod": "vite build"
     },
     "devDependencies": {
-        "@appbaseio/reactivesearch-vue": "https://gitpkg.now.sh/api/pkg?url=royduin/reactivesearch/packages/vue&commit=patch-4&scripts.postinstall=yarn%20install%20--ignore-scripts%20%26%26%20yarn%20run%20build-es&scripts.build-es=nps%20build.es",
-        "@babel/core": "^7.18.10",
-        "@babel/plugin-proposal-object-rest-spread": "^7.18.9",
-        "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-        "@babel/plugin-transform-runtime": "^7.18.10",
-        "@babel/preset-env": "^7.18.10",
+        "@appbaseio/reactivesearch-vue": "^1.27.0 <1.33.0",
```
[the remark this answers is not in the document]
Comment (indykoning) [reply package.json]: 1.33.0 introduced a bug making the library unusable in Rapidez
https://github.com/appbaseio/reactivesearch/issues/2120
This has been fixed from 1.33.1 onwards though 🙂

**Pull request merged**